api/scripts/wazuh-apid.py

```diff
@@ -181,23 +181,32 @@ def exit_handler(signum, frame):
     from api import alogging, configuration
     from wazuh.core import common, utils
 
-
     def set_logging(log_path='logs/api.log', foreground_mode=False, debug_mode='info'):
         for logger_name in ('connexion.aiohttp_app', 'connexion.apis.aiohttp_api', 'wazuh-api'):
             api_logger = alogging.APILogger(
                 log_path=log_path, foreground_mode=foreground_mode, logger_name=logger_name,
                 debug_level='info' if logger_name != 'wazuh-api' and debug_mode != 'debug2' else debug_mode
             )
             api_logger.setup_logger()
-
+        if os.path.exists(log_path):
```

Comment:
Have you tested this? I understand that now, this has to be done for both log and json files. But, we used to do this with the absolute path to the file and now you are using a relative path.

Old code:

```python
# Set correct permissions on api.log file
    if os.path.exists(os.path.join(common.wazuh_path, API_LOG_FILE_PATH)):
        os.chown(os.path.join(common.wazuh_path, API_LOG_FILE_PATH), common.wazuh_uid(), common.wazuh_gid())
        os.chmod(os.path.join(common.wazuh_path, API_LOG_FILE_PATH), 0o660)
```

Reply:
There was an error in the old code that I fixed. `API_LOG_FILE_PATH` was the full path of the `api.log` file. Therefore the `os.path.join` is redundant.

However, the default value of `log_path` at `set_logging` should be the full path.